pkg/k8shandler/migrations/kibana5to6.go

```diff
@@ -25,7 +25,6 @@ func (mr *migrationRequest) reIndexKibana5to6() error {
 	}
 
 	if mr.migrationCompleted() {
-		log.Info("migration completed: re-indexing", "from", kibanaIndex, "to", kibana6Index)
```

Comment:
This seems like a useful info log.  do we log it elsewhere?

Reply:
this is exceptionally noisy and even shows up when not migrating.